packages/paste-core/components/select/src/Select.tsx

```diff
@@ -102,7 +107,7 @@ const Select = React.forwardRef<HTMLSelectElement, SelectProps>(
             size={multiple ? size : 0}
             variant={variant}
           >
-            {children}
+            {showOptions && children}
```

Comment:
question: should this have it's own changeset? I know its for hydration, but if someone upgrades would it be useful to us or them to have a call out specifically in this components changelog mentioning this change?

Reply:
Yeah a changeset could be helpful for this!